pkg/network/tracer/connection/kprobe/tracer.go

```diff
@@ -303,18 +307,15 @@ func (t *kprobeTracer) Stop() {
 
 func (t *kprobeTracer) GetMap(name string) *ebpf.Map {
 	switch name {
+	case string(probes.ConnectionStatesMap):
```

Comment:
Do you need the switch statement anymore?

Reply:
the switch is used as a filter to all the real ebpf.GetMap()